### PR TITLE
allow configuration of puppet wsgi processes

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -99,6 +99,8 @@
   $max_keep_alive = 10000
   $num_workers = min($::processorcount, 8)
 
+  $puppet_wsgi_processes = 3
```
Comment (ekohl) [inline manifests/params.pp]: Any particular reason to pick 3 here?

Comment (jlsherrill) [inline manifests/params.pp]: its the default value  pulp provides:  https://github.com/pulp/pulp_puppet/blob/master/pulp_puppet_plugins/etc/httpd/conf.d/pulp_puppet.conf#L39

File: manifests/init.pp [hunk 2]
```diff
@@ -254,6 +254,8 @@
 # $max_keep_alive::             Configuration value for apache MaxKeepAliveRequests
 #                               type:integer
 #
+# $puppet_wsgi_processes::      Number of WSGI processes to spawn for the puppet webapp
```
Comment (ekohl) [inline manifests/init.pp]: It needs to be included in the header below as well

Comment (jlsherrill) [inline manifests/init.pp]: doh!  thanks, updated

**Comment by ekohl** [comment]
ACK on green tests

File: manifests/params.pp (hunk 1 above)
Comment (ekohl) [inline manifests/params.pp]: I noticed that after I made the comment. That's fine then, if needed it can be tuned later like num_workers above.

**Comment by jlsherrill** [comment]
thanks @ekohl !

**Pull request merged**